### PR TITLE
AngularV19

### PR DESCRIPTION
**Pull request by JeevanMahesha**

### CODE REVIEW

**Comment by JeevanMahesha** [comment]
@JeanMeche I have updated it for V19 can you please review.

File: angular/package.json [hunk 1]
```diff
@@ -7,21 +7,21 @@
...
+    "zone.js": "~0.15.0"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^18.1.0",
-    "@angular/cli": "^18.1.0",
-    "@angular/compiler-cli": "^18.1.0",
+    "@angular/build": "^19.0.0",
+    "@angular/cli": "^19.0.0",
+    "@angular/compiler-cli": "^19.0.0",
     "typescript": "~5.5.0"
```
Comment (JeanMeche) [inline angular/package.json:25]: Can you also bump TS to 5.6, ty !

File: angular/tsconfig.json [hunk 2]
```diff
@@ -12,7 +12,6 @@
     "esModuleInterop": true,
     "sourceMap": true,
     "declaration": false,
-    "downlevelIteration": true,
     "experimentalDecorators": true,
```
Comment (JeanMeche) [inline angular/tsconfig.json:15]: can we also add `strictStandalone: true` ?

**Review by JeanMeche — APPROVED** [review]
LGTM !

**Review by d3lm — APPROVED** [review]

**Comment by d3lm** [comment]
@JeevanMahesha Seems like `prettier` check fails.

**Comment by d3lm** [comment]
Thanks @JeevanMahesha 🙏

**Pull request merged**